typescript/sdk/src/token/Token.test.ts

```diff
@@ -168,18 +171,32 @@ describe('Token', () => {
         PROTOCOL_TO_ADDRESS[token.protocol];
       if (!address)
         throw new Error(`No address for standard ${tokenArgs.standard}`);
+
+      const sandbox = sinon.createSandbox();
+      sandbox.stub(multiProvider, 'getEthersV5Provider').returns({
+        getBalance: async () => '100',
+      } as any);
+      sandbox.stub(multiProvider, 'getCosmJsProvider').returns({
+        getBalance: async () => ({ amount: '100' }),
+      } as any);
+      sandbox.stub(multiProvider, 'getCosmJsWasmProvider').returns({
+        getBalance: async () => ({ amount: '100' }),
+        queryContractSmart: async () => ({
+          type: { native: { fungible: { denom: 'denom' } } },
+        }),
+      } as any);
+      sandbox.stub(multiProvider, 'getSolanaWeb3Provider').returns({
+        getBalance: async () => '100',
+        getTokenAccountBalance: async () => ({ value: { amount: '100' } }),
+      } as any);
```

Comment:
these are nice
I wonder if we can export this in some test util

Reply:
will do